README.rst

```diff
@@ -211,6 +211,30 @@ Get single message example:
 
 There are more examples and the RabbitMQ tutorial in the `documentation`_.
 
+See also
+==========
+
+`Propan`_:fire:
+------
+
+**Propan** is a powerful and easy-to-use Python framework for building event-driven applications that interact with any MQ Broker.
+
+If you need no deep dive into **RabbitMQ** details, you can use more high-level **Propan** interfaces:
+
+.. code-block:: python
+
+   from propane import Propaneapp, Rabbit Broker
```

Comment:
`RabbitBroker` I guess. The same in the file below.

Reply:
Sure, thx